src/rideAxis/specifications/index.md

```diff
@@ -0,0 +1,27 @@
+# Ride Axis
```

Comment:
Since it's not part of the design system, should we put this kind of components in another bucket?
For example we discussed having `Internal` suffix when a component should only be used in ui-library, maybe we should have a way to identify components that we created because it's easier on the technical side. Another approach would be to ask designers to add this component in the design system. Wdyt?

Reply:
I'll move to '_utils' for now

Reply:
I'm afraid suffixing all private symbols (components, hooks, functions, constants, etc.) will be too cumbersome.
I simple solution can be to use an __internal/_ folder.

Reply:
Yeah.. that's why I'm moving the component there and creating a "Internal" Section on Storybook taxonomy